rmf_fleet_adapter/src/rmf_fleet_adapter/tasks/Patrol.cpp

```diff
@@ -67,8 +69,20 @@ void add_patrol(
             std::make_move_iterator(place.errors.begin()),
             std::make_move_iterator(place.errors.end()));
         }
-        auto desc = GoToPlace::Description::make_with_multiple(goals);
+        auto desc = GoToPlace::Description::make_for_one_of(goals);
         return {desc, errors};
```

Comment:
```suggestion
```

Reply:
Move this line below, otherwise the description would return before accounting for the constraints